src/main.py

```diff
@@ -2859,7 +2860,7 @@ def post_all(self, o):
         quicklinks = post["quicklinksid"]
         quickreports = post["quickreportsid"]
         quickreportscfg = []
-        reports = asm3.reports.get_reports(o.dbo)
+        reports = asm3.reports.Mail(o.dbo)
```

Comment:
What is Mail? It's not a member of reports.py and it doesn't seem to be in your changes? I'm not sure this change is relevant to this ticket either since this bit of code is updating quick reports